docs/conf-sessions.rst

```diff
@@ -1035,6 +1035,27 @@ Example with MQTT as a session type:
 
 Note that if a ``client_id`` is omitted upon connecting Tsung will create a random one, prefixed with ``tsung-``.
 
+Message stamping
+""""""""""""""""
+
+It is possible to stamp measure the broker's forwarding latency by setting
+``stamped`` attribute inside ``<publish>`` element to ``true``. The stamp
+will include current timestamp and ID of the sender node. If the recipient
+will recognize the node ID, it will compare the timestamp inside message
+with the current one. The difference will be reported as ``mqtt_forward_latency``
+metric (in milliseconds). The aim of node ID comparison is to avoid slight
+inconsistencies of timestamps on different Tsung nodes.
+
+Only a fraction of requests will hit the same node they originated from,
+but with request rate high enough this fraction should be sufficient.
+
+``stamped`` is allowed only with ``size`` attribute. ``data`` will cause
+``stamped`` to be ignored. There is a minimal length of the stamp,
+roughly 30 bytes. When ``size`` is greater than stamp length, random
+padding will be added to the stamp. If the stamp length is higher than
+``size``, then only stamp will be used as messagecontent, effectively
```

Comment:
typo -> `message content`

Also, maybe we can add a short example with `stamped="true"`? I'm not that familiar with MQTT, but reading this, I'm not sure what the `size` attribute is. `data` I assume is the message content?

Reply:
yes, “messagecontet”  and  ``size`` was copied from line 301 in this file, and ``size`` is the attribute for Jabber. I would remove ``size``.
``stamped`` is just a switch to enable/disable the time stamp feature, and the time stamp feature is independent with other mqtt feature, so I think the original MQTT example is enough.